### PR TITLE
Upgrade Valkyrie v2

### PR DESCRIPTION
**Pull request by mburns**
- [ ] Update Java https://github.com/racker/chef/pull/3731 (?)
- [x] Remove username and password from valkyrie config
- [x] upgrade repose to 7.3.6.0
- [x] update endpoint URL
 * was `https://valkyrie.prod.ord1.us.ccp.rackspace.net/`, `http://valkyrie-n01.staging.ord1.us.ccp.rackspace.net:3000/`
 * now `https://staging.api.valkyrie.rackspace.com` `https://api.valkyrie.rackspace.com`
- [ ] ~~update version to `1.3-SNAPSHOT` (?)~~
- [ ] ~~build new artifacts, download from CircleCI, update to Cloud Files~~
- [ ] chef.git PR to update Berksfile, valkyrie templates https://github.com/racker/chef/pull/3768

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -118,7 +113,7 @@
   node.set['repose']['endpoints'] = [{
     cluster_id: 'repose',
     id: 'public_api',
-    protocol: 'http',
+    protocol: 'https',
```
Comment (drenalin23) [inline recipes/default.rb]: Why are you changing this to https?  Does this host support https on localhost?

File: recipes/default.rb [hunk 2]
```diff
@@ -89,17 +88,13 @@
 
   identity_url = URI.join(identity_url, '/').to_s # strip trailing path (repose adds it)
 
-  valkyrie_url = repose_credentials["valkyrie_url_#{node.chef_environment}"]
-  valkyrie_username = repose_credentials["valkyrie_username_#{node.chef_environment}"]
-  valkyrie_password = repose_credentials["valkyrie_password_#{node.chef_environment}"]
+  valkyrie_url = node.chef_environment == 'prod' ? 'https://api.valkyrie.rackspace.com ' : 'https://staging.api.valkyrie.rackspace.com'
```
Comment (drenalin23) [inline recipes/default.rb]: Is there a need for the space after api.valkyrie.rackspace.com?

**Comment by drenalin23** [comment]
Couple of comments - but looks good to me once those are checked.

**Comment by mburns** [comment]
Thanks @drenalin23. Both notes were things I didn't want in this PR. removed.

**Comment by drenalin23** [comment]
+1

**Comment by mburns** [comment]
https://gist.github.com/mburns/fd6fa9c3f3ff112a520e61183816380b

**Pull request merged**